### DISCUSSION TITLE
Pushing to repository

### DISCUSSION THREAD

**Discussion by JohnUja** [community/community#56732] [Repositories]
Hello I have been trying to push changes to my remote repo for the past hour, I have  staged and committed the changes and have pushed it to the repo, no errors seem to be occurring and it shows a status that everything uploaded correctly, however when I refresh my repo I cant see the changes. I've checked the git remote -v and my local and remote repositories seem to be linking correctly, however when I check git status, I get the message that my branch is ahead of main by 1 commit and it says I should fix by a git push which I already did. I am thinking of manually updating the repo by uploading the files my self, but I am worried the changes might push later and cause conflict

**Comment by odessaesconde18** [comment]
"WATTA good job!men!"

**Comment by tuves** [comment]
Welcome to the GitHub Community, @JohnUja, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
